dynatrace_collector/dynatrace-collector

```diff
@@ -1,8 +1,8 @@
 #!/bin/bash
 # The init.d script for the wavefront collector tools.
 
-PID=/tmp/wavefront/dynatrace/pid/dcollector.pid
-OUT=/tmp/wavefront/dynatrace/log/dynatrace.log
+PID=/var/run/dtcollector.pid
+OUT=/var/log/wavefront/dynatrace/log/dynatrace.log
```

Comment:
Seems redundant. It's already under `/var/log`. Maybe just `/var/log/wavefront/dynatrace/dynatrace.log`?